flow360/examples/tutorial_BETDisk.py

```diff
@@ -0,0 +1,15 @@
+"""
+Tutorial_BETDisk meshing example
+"""
+
+from .base_test_case import BaseTestCase
+
+
+class Tutorial_BETDisk(BaseTestCase):
```

Comment:
this is not valid class name. Should be PascalCase:
```suggestion
class TutorialBETDisk(BaseTestCase):
```
rename all occurrences

Reply:
Thank you, I will correct that